compiler/rustc_const_eval/src/interpret/machine.rs

```diff
@@ -191,22 +192,17 @@ pub trait Machine<'mir, 'tcx: 'mir>: Sized {
     /// nor just jump to `ret`, but instead push their own stack frame.)
     /// Passing `dest`and `ret` in the same `Option` proved very annoying when only one of them
     /// was used.
-    fn find_mir_or_eval_fn(
+    fn find_mir_or_extra_fn(
```

Comment:
The documentation of this function needs an update

Reply:
I fixed it up, should be better now.

Athough it looks like there are other tests and comments mentioning `__rust_maybe_catch_panic`, which doesn't seem to exist anymore?... Possibly someone needs to go over them and update them too.